train-tracker/src/Map.js

```diff
@@ -20,20 +20,20 @@ L.Icon.Default.mergeOptions({
     shadowUrl: markerShadow,
 });
 
-const Map = () => {
+const Map = ({trains}) => {
     const [railLines, setRailLines] = useState(null);
     const [stations, setStations] = useState(null);
-    const [trains, setTrains] = useState([]);
+    // const [trains, setTrains] = useState([]);
```

Comment:
can we remove these or will we need them in the future?

Reply:
i feel like we don't want any state in the map, just want it all coming down from app